packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -262,7 +301,7 @@ describe(`onRender`, () => {
     // TODO: unstable_now is called by more places than just the profiler.
     // Rewrite this test so it's less fragile.
     if (gate(flags => flags.enableDeferRootSchedulingToMicrotask)) {
-      assertLog(['read current time', 'read current time']);
+      assertLog(['read current time']);
```

Comment:
When using `ReactDOMClient.createRoot()`, this is `['read current time', 'read current time', 'read current time', read current time']`. With `ReactDOM.render`, it's `['read current time']`

Can someone help me understand what might be causing this to be different between RTR, cR, and render?